kdwsdl2cpp/src/main.cpp

```diff
@@ -163,19 +221,52 @@ int main(int argc, char **argv)
         return 1;
     }
 
+    // if you're saying "just make the impl-file", you can't
+    //    also say "make both the header and the impl"
+    if (both && impl) {
```

Comment:
Same if specifying -o and -both, right? This should give an error too.
To do it here you'd need another bool, the alternative would be to test it when parsing -both (outputFile should still be empty) and when parsing -o (both should be false).